test/unit/redux.test.js

```diff
@@ -1,98 +1,231 @@
+      name: 'Snickers',
+      chocolate: 'yes',
+      nuts: 'yes',
+      fruity: 'no',
+      delicious: 'yes'
     },
     {
-      name: "Almond Joy",
-      chocolate: "yes",
-      nuts: "yes",
-      fruity: "no",
-      delicious: "yes"
+      name: 'Almond Joy',
+      chocolate: 'yes',
+      nuts: 'yes',
+      fruity: 'no',
+      delicious: 'yes'
     },
     {
-      name: "Black Licorice",
-      chocolate: "no",
-      nuts: "no",
-      fruity: "no",
-      delicious: "no"
+      name: 'Black Licorice',
+      chocolate: 'no',
+      nuts: 'no',
+      fruity: 'no',
+      delicious: 'no'
     },
     {
-      name: "Skittles",
-      chocolate: "no",
-      nuts: "no",
-      fruity: "yes",
-      delicious: "yes"
+      name: 'Skittles',
+      chocolate: 'no',
+      nuts: 'no',
+      fruity: 'yes',
+      delicious: 'yes'
     }
   ],
-  labelColumn: "delicious",
-  selectedFeatures: ["chocolate", "nuts"],
+  labelColumn: 'delicious',
+  selectedFeatures: ['chocolate', 'nuts'],
   columnsByDataType: {
-    chocolate: "categorical",
-    nuts: "categorical",
-    delicious: "categorical"
+    chocolate: 'categorical',
+    nuts: 'categorical',
+    delicious: 'categorical'
   }
 };
 
 const resultsState = {
   data: [
     {
-      sun: "high",
+      sun: 'high',
       height: 3.8
     },
     {
-      sun: "high",
+      sun: 'high',
       height: 3.9
     },
     {
-      sun: "medium",
+      sun: 'medium',
       height: 2.6
     },
     {
-      sun: "medium",
+      sun: 'medium',
       height: 2.5
     },
     {
-      sun: "low",
+      sun: 'low',
       height: 0.9
     },
     {
-      sun: "low",
+      sun: 'low',
       height: 1.6
     }
   ],
-  labelColumn: "height",
+  labelColumn: 'height',
+  labelColumnCategorical: 'sun',
   accuracyCheckPredictedLabels: [4.0, 3.75, 2.63, 2.46, 1.6, 1.0],
   accuracyCheckLabels: [3.9, 3.8, 2.6, 2.5, 1.6, 0.9]
 };
 
-describe("redux functions", () => {
-  test("getAccuracyRegression", async () => {
+const resultsStateClassification = {
+  data: [
+    {
+      weather: 'sunny',
+      temp: 'hot',
+      play: 'no'
+    },
+    {
+      weather: 'overcast',
+      temp: 'hot',
+      play: 'yes'
+    },
+    {
+      weather: 'overcast',
+      temp: 'mild',
+      play: 'yes'
+    },
+    {
+      weather: 'overcast',
+      temp: 'cool',
+      play: 'yes'
+    },
+    {
+      weather: 'rainy',
+      temp: 'mild',
+      play: 'yes'
+    },
+    {
+      weather: 'rainy',
+      temp: 'cool',
+      play: 'no'
+    }
+  ],
+  labelColumn: 'play',
+  selectedFeatures: ['temp', 'weather'],
+  columnsByDataType: {
+    weather: 'categorical',
+    temp: 'categorical',
+    play: 'categorical'
+  },
+  accuracyCheckLabels: ['no', 'yes', 'yes', 'yes', 'yes', 'no']
+};
+
+describe('redux functions', () => {
+  test('getAccuracyClassification - all accurate', async () => {
+    const mockState = resultsStateClassification;
+    mockState.accuracyCheckPredictedLabels = [
```

Comment:
Can you set your test inputs to named variables, please? Something like `const accurateResults = ['no', 'yes', 'yes', 'yes', 'yes', 'no']` then `resultsStateClassification.accuracyCheckPredictedLabels = accurateResults;`. You'd follow that pattern for each test case. I think it will increase readability, allow you to remove `mockState` and make it more clear what results to expect in each case.

Reply:
Yes that makes sense and I agree it will increase readability! I'll double check on the styling changes....perhaps I forgot to push them...hopefully that's the case :) I won't be able to work on it today but I will plan to push changes before  the start of next week.

Reply:
Awesome! This change looks great. Thank you!